pegasus/helper_modules/dashboard.rb

```diff
@@ -14,6 +14,10 @@ def self.admin?(user_id)
     !!db[:users][id: user_id, admin: true]
   end
 
+  def self.hidden_script_access?(user_id)
+    admin?(user_id) || User.get(user_id).has_permission?('hidden_script_access')
```

Comment:
Not sure if this can happen given the current call path, but we might want to protect against a bad user_id. The old [admin?](https://github.com/code-dot-org/code-dot-org/blob/16cbeb9f0c6c46bdd8fb142c58a2832a34c89544/pegasus/helper_modules/dashboard.rb#L13-L15) call will simply return false if user_id is invalid, but the new code will fail trying to access nil User.

What about something like this?
```ruby
user = User.get(user_id)
user && (user.admin? || user.has_permission?('hidden_script_access'))
```